IBX-6626: Simplify user actions (#233)
ibexa/documentation-user: docs/content_management/content_organization/copy_move_hide_content.md

You can remove content by clicking **Send to Trash** in the menu.
If you remove a Content item that has children (other content under it in the content tree),

> both this item and the children are removed. This also breaks the connection between the items,
> so it's impossible to restore them with the same structure.

Author: ```suggestion
both this item and the children will be removed. This also breaks the connection between the items,
so you will not be able to restore them with the same structure.
```

![Warning before emptying the trash](img/empty_trash_warning.png "Warning before emptying the Trash")

> If a Content item has more than one Location, selecting **Send to Trash** removes the Content item only from the current Location.
> The content appears in Trash only once you have removed the last Location.

Author: ```suggestion
If a Content item has more than one Location, selecting **Send to Trash** will remove the Content item only from the current Location.
The content will appear in Trash only once you have removed the last Location.
```

## Multi-file delete

> To delete multiple items, select them and click **Delete selected items** in the **Sub-items** top bar. Confirm your choice in the pop-up window with the **Send to Trash** button. All selected files are moved to trash.

Author: ```suggestion
To delete multiple items, select them and click **Delete selected items** in the **Sub-items** top bar. Confirm your choice in the pop-up window with the **Send to Trash** button. All selected files will be moved to trash.
```

![Hide content icon](img/hide_content_icon.png)

When you click **Hide**, you can choose to **Hide later**

> and select and date and time when the Content item is hidden:

Author: ```suggestion
and select and date and time when the Content item will be hidden:
```
